src/main/default/lwc/ratingUtils/ratingUtils.js

```diff
@@ -14,12 +14,13 @@ const DOCUMENT_ID_FIELD = 'Document_id__c';
 export async function incrementScore(id) {
   try {
     const response = await getRating({ documentId: id });
-    if (response) {
-      this.object = response;
+    if (response && response.length > 0) {
+      this.object = response[0];
```

Comment:
In my opinion, `getRating` should return a single rating object, not a list. The SOQL query is an implementation detail that should be scoped to `getRating` only. An implementer calling `getRating` should not have to know how the rating is retrieved. The method should simply return the rating if it exists, or a default rating.

Reply:
Nevermind, I see this was fixed in another changeset.